src/tup/server/fuse_fs.c

```diff
@@ -39,7 +39,7 @@
 #include <sys/types.h>
 #include <sys/resource.h>
 
-#if defined(__FreeBSD__)
+#if defined(__FreeBSD__) || defined(__linux__)
```

Comment:
Since it doesn't seem like using AT_SYMLINK_NOFOLLOW in fuse_fs.c has any impact, I think we should just remove the access_flags altogether and call faccessat(..., 0) wherever it gets used.

Reply:
Done.